cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewJob.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2025 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.app.preview;
+
+/**
+ * An object that holds runner information linked to the {@link PreviewRequest}.
+ */
+public class PreviewJob {
+
+  private final PreviewRequest previewRequest;
+  private final byte[] runnerInfo;
```

Comment:
Can `byte[] runnerInfo` field be added in the `PreviewRequest` class instead of creating a new class `PreviewJob`?